db_store/src/settings.rs

```diff
@@ -11,7 +11,7 @@ pub enum AuthType {
 
 #[derive(Debug, Deserialize, Clone)]
 pub struct Settings {
-    pub max_connections: u32,
+    pub connections: u32,
```

Comment:
I understand the reasoning for this change, but I would prefer we keep our settings descriptive so we can easily understand their purpose.